gallery/src/components/NxGlobalSidebar/NxGlobalSidebarPage.tsx

```diff
@@ -0,0 +1,226 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import CodeExample from '../../CodeExample';
+import { NxP, NxCode, NxTable } from '@sonatype/react-shared-components';
+
+import './NxGlobalSidebarPage.scss';
+
+const NxGlobalSidebarExample = require('./NxGlobalSidebarExample.tsx?raw'),
+    NxGlobalSidebarWithPageSidebarExample = require('./NxGlobalSidebarWithPageSidebarExample.tsx?raw');
+
+export default function NxGlobalSidebarPage() {
+  return (
+    <>
+      <section className="nx-tile">
+        <header className="nx-tile-header">
+          <div className="nx-tile-header__title">
+            <h2 className="nx-h2">Description</h2>
+          </div>
+        </header>
+        <div className="nx-tile-content">
+          <NxP>
+            <NxCode>NxGlobalSidebar</NxCode> is a collapsible page level sidebar. It has two main sections:
+          </NxP>
+          <ul className="nx-list nx-list--bulleted">
+            <li className="nx-list__item">A header which contains branding as well as the open/close toggle</li>
+            <li className="nx-list__item">
+              A navigation link section which display an icon plus text in the open state and an icon in the closed
+              state
+            </li>
+          </ul>
+          <NxP>
+            Because <NxCode>NxGlobalSidebar</NxCode> contains product branding it should not be used in conjunction
+            with <NxCode>NxPageHeader</NxCode> or <NxCode>NxNexusPageHeader</NxCode>. Instead it should be paired with
+            <NxCode>NxSystemHeader</NxCode> (not yet added to the gallery).
```

Comment:
This isn't what I ended up naming it in my branch.  We should probably meet with @dturner32 and decide what the best name is

Reply:
> This isn't what I ended up naming it in my branch. We should probably meet with @dturner32 and decide what the best name is

I was also wondering if it should be pulled until your work is merged.

Reply:
> I was also wondering if it should be pulled until your work is merged.

I'm fine with that

Reply:
> > I was also wondering if it should be pulled until your work is merged.
> 
> I'm fine with that

https://github.com/sonatype/sonatype-react-shared-components/pull/371/commits/ad49c6917bc88d132ffe8a9b98b717d7fb4fb80a